test/hummingbot/connector/exchange/trading212/test_trading212_exchange.py

```
@@ -0,0 +1,32 @@
+import asyncio
```

Flake8 (7.2.0)

[error] 'asyncio' imported but unused

(F401)